### ISSUE TITLE
Items drop doesn't work with a particular iframe (videochat)

### ISSUE THREAD

**Issue by andtos90** [react-dnd/react-dnd#2184]
**Describe the bug**
Items drop doesn't work with a particular iframe. This behaviour happens when the iframe with a video chat library is rendered below the droppable target (target has a greater z-index than the iframe). 

**Reproduction**
**You need to enter a random generated chat room to test this behaviour**, please don't give audio/video permission to the tab if you're not comfortable with this.

[Live Reproduction](https://codesandbox.io/s/react-dnd-not-working-psvwg)

1. Open the Codesandbox preview in a new window
2. Try to drag "glass" to the box with "Drag a box here" label
3. Drop the item

**Expected behavior**
An alert should open with this text: `You dropped Glass into Dustbin!`. 

**Desktop (please complete the following information):**

- OS: OSX Catalina
- Browser: Chrome
- Version: 80

**Additional context**
The dnd seems to work fine with the embedded preview

**Comment by dhiraj-indusved** [comment]
I'm facing the exact same issue, where I have an iframe just underneath the dropabble target. When the item is dragged I increase the z-index of the dropabble target. The app works if run locally with safari, firefox and chrome. However if I build the app and deploy it to server it works only on Firefox and Safari. 

Initially I thought the issue is because of the delay in drop and in mean time the droppable targets gets behind the iframe as soon as I drop the item and hence the issue. However I tried giving a delay to the method that puts droppable target behind the iframe but it still have this same issue on chrome browser in production build.

I'm still investigating what might be the cause. Once I figure out something I'll update

**Comment by CodeDem** [comment]
@andtos90 I was able to fix it in my setup using `pointer:events: none` on Iframe when dragging starts and passing the data to iframe  using `window.postMessage()` API when the element is dropped on a droppable target. This issue seems to be there on chrome due to two different origins.

**Comment by andtos90** [comment]
Thanks @CodeDem, it's a good workaround. I'll leave the issue opened because it's still not completely solved

**Comment by CodeDem** [comment]
Yeah, it's a workaround. Not sure when chrome will do something about it :(
